<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MdrFwUart</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QPushButton" name="touchMCUPB">
      <property name="text">
       <string>Start</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="openFilePB">
      <property name="text">
       <string>Файл прошивки</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="fwfnLE">
      <property name="text">
       <string>E:\arm\61.10.15.01\Objects\u61101501dd2.hex</string>
      </property>
      <property name="placeholderText">
       <string>Выберите файл прошивки</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="portBaudrateCB">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>230400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>460800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>921600</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QComboBox" name="portNameCB"/>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="loadCB">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="eraseCB">
      <property name="text">
       <string>Erase</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="programCB">
      <property name="text">
       <string>Program</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="verifyCB">
      <property name="text">
       <string>Verify</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QCheckBox" name="runCB">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="dumpfnPB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Файл дампа</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="4">
     <widget class="QLineEdit" name="dumpfnLE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите имя файла для сохранения прошивки</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="flasherfnPB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Файл флешера</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="4">
     <widget class="QLineEdit" name="flasherfnLE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Оставьте поле пустым для использования стандартного  флешера</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="4">
     <widget class="QLabel" name="sL">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Ход работы</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="5">
     <widget class="QPlainTextEdit" name="t">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QGroupBox" name="mcuSelectGB">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Выбирите микроконтроллер</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="mcu9xRB">
         <property name="text">
          <string>1986ВЕ9x, 1901ВЦ1Т</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mcu1RB">
         <property name="text">
          <string>1986ВЕ1, ВЕ3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mcu4RB">
         <property name="text">
          <string>1986ВЕ4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_Qt"/>
    <addaction name="about"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Qt">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
